themes/doks/layouts/index.html

```diff
@@ -109,7 +109,7 @@ <h3 class="card-title">API Reference</h3>
             <p>Javadocs and KDocs for all publicly-exposed APIs in the Corda 5 and Corda 4 series: Corda OS, Corda Enterprise, and Corda Enterprise Network Manager (CENM).</p>
           </div>
           <div class="card-footer">
-            <a href="en/api-ref.html" class="btn rounded external">API references</a>
+            <a href="en/api-ref.html" class="btn rounded stretched-link">API References</a>
```

Comment:
The other labels all have verbs.. Should this be "Read API References" or similar?